pymodels/__init__.py

```diff
@@ -8,16 +8,17 @@
 from . import TS_V04_01
 from . import SI_V24_04
```

Comment:
I think we can delete the folder `SI_V24_04` by now, can't we?

Reply:
actually, I think we don't need to keep old accelerator versions, as we did in matlab. they are in code version control, anyway, and it is easy to restore them, if needed. what do you think, @fernandohds564 , @murilobalves ?

Reply:
I agree @xresende !

Reply:
ok. I'll add a commit.